### ISSUE TITLE
Clone dotCMS auth to QA

### ISSUE THREAD

**Issue by sfreudenthaler** [dotCMS/core#30662]
[Source Slack Message](https://dotcms.slack.com/archives/CQMHM7PNJ/p1731598081511149)

Now that dotCMS.com is treated like a real customer, we can't just go firing random things there to test.  Seems like there are gaps in what we have on the QA servers.  I'm proposing an approach where we'd keep a relatively recent clone of dotcms.com instance in a QA environment so devs could test things like headless, hybrid, UVE, and modern FE implimentation.

**Comment by john-thomas-dotcms** [comment]
@sfreudenthaler did we implement this with the creation of the staging server, or is this something different?

**Comment by sfreudenthaler** [comment]
Yeah I think we’re good. For record keeping we used push publishing with force flag to do it on-demand. There is separate scope if we wanted to build something that kept it up to date or a read replica kind of thing. On Feb 27, 2025, at 09:41, John Michael Thomas ***@***.***> wrote:﻿
  @sfreudenthaler did we implement this with the creation of the staging server, or is this something different?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



john-thomas-dotcms left a comment (dotCMS/core#30662)
@sfreudenthaler did we implement this with the creation of the staging server, or is this something different?

**Comment by john-thomas-dotcms** [comment]
Completed
